android/app/src/main/java/io/mosip/residentapp/RNVCVerifierModule.java

```diff
@@ -21,6 +25,13 @@ public String getName() {
 
     @ReactMethod
     public void verifyCredentials(String vc, Promise promise) {
-        promise.resolve(credentialsVerifier.verifyCredentials(vc));
+
+        VerificationResult result = credentialsVerifier.verify(vc, CredentialFormat.LDP_VC);
```

Comment:
This should not be hardcoded as LDP_VC. verifyCredentials function should accept format from caller
@KiruthikaJeyashankar please check it

Reply:
We had a discussion on this last Tuesday and Bala was working towards it, may be he has some changes in local. Need to check it out

Reply:
Updated to pass credentialFormat